modules/osd-aws-privatelink-config-dns-forwarding.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-aws-privatelink-creating-cluster.adoc
+
+[id="osd-aws-privatelink-config-dns-forwarding_{context}"]
+= Configuring AWS PrivateLink DNS forwarding
+
+With AWS PrivateLink clusters, only a private hosted zone is created in Route53. With a private hosted zone, records within the zone are resolvable only from within the VPC to which it is assigned.
+For more information about private hosted zones, see link:https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zones-private.html[AWS private hosted zones documentation].
+
+.Prerequisites
+
+* Your corporate network or other VPC has connectivity
+* Ports UDP/53 and TCP/53 ARE enabled across your networks to allow for DNS queries
```

Comment:
For searchability I would recommend and optional edit here: 

UDP port 53 and TCP port 53 are enabled across your networks to allow for DNS queries